src/canvas/static_canvas.class.ts

```diff
@@ -1296,9 +1298,9 @@ export class StaticCanvas<
   createSVGRefElementsMarkup(): string {
     return ['background', 'overlay']
```

Comment:
```suggestion
    return (['background', 'overlay'] as const)
```

will type the array and remove the need for the extra as 'overlayColor' | `backgroundColor`

Reply:
6b10575